maintainers/team-list.nix

```diff
@@ -816,11 +816,14 @@ with lib.maintainers;
     shortName = "Mercury Employees";
   };
 
+  # same as https://github.com/orgs/NixOS/teams/nix-team
   nix = {
     members = [
       eelco
-      grahamc
```

Comment:
cc @nbp and @grahamc I dropped you because you seem not to maintain nix in nixpkgs and you are not in the nix team.